### ISSUE TITLE
How to use parameter seed to control the training of xgboost?

### ISSUE THREAD

**Issue by Liumucan** [dmlc/xgboost#7496]
I want to use the parameter _seed_ to control the randomness of xgboost. I thought I can use different seed to obtain different prediction results with xgboost with the same parameters and the same dataset (just like random_state in sklearn API), but it seems that adjusting the parameter _seed_ doesn't change the predictive results. So is there something wrong about the way I use the parameter _seed_ ? How can I use _seed_ to obtain different results for the same model?
Here is part of my codes:
--------------------------------------------------------------
params = {
        'silent': 1,
        'objective': 'binary:logistic',
        'gamma': 0.1,
        'min_child_weight': 5,
        'disable_default_eval_metric': 1,
        }
model = xgb.train(params=params,
                      dtrain=d_train,
                      num_boost_round=200,
                      feval=AUROC,
                      evals=[(d_train, 'dtrain'), (d_val, 'dval')],
                      obj=binary_cross_entropy)
--------------------------------------------------------------
Thanks!

**Comment by trivialfis** [comment]
Seed is only useful if you enable sampling duru training.

**Comment by Liumucan** [comment]
Thank you very much!
